### PR TITLE
Add Put as a thin wrapper around raw put

### PR DESCRIPTION
**Pull request by bakins**
I have a couple of use cases where I have to currently decide which function to call (Set vs Update vs Create vs CompareAndSwap) where just conditionally setting the options to PUT is much cleaner.

### CODE REVIEW

File: etcd/put.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package etcd
+
+// Put issues a PUT request
+func (c *Client) Put(key string, value string, ttl uint64,
+	options Options) (*Response, error) {
```
Comment (kelseyhightower) [inline etcd/put.go]: Can you put this on one line. We are not really worried about 80 columns.

**Comment by kelseyhightower** [comment]
@bakins Can you make that one minor change?

**Comment by kelseyhightower** [comment]
LGTM /cc @yichengq

**Comment by yichengq** [comment]
based on discussion OOL, we agree that go-etcd should provide actions instead of exposing HTTP methods.

**Pull request closed without merging**